[WARN ] 2018-04-26 13:09:14 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server 10.65.1.16/10.65.1.16:2181, unexpected error, closing socket connection and attempting reconnect
     java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:09:14 org.I0Itec.zkclient.ZkClient@(ZkClient.java:449):zookeeper state changed (Disconnected)
     [INFO ] 2018-04-26 13:09:16 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [INFO ] 2018-04-26 13:09:16 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:876):Socket connection established to 10.65.1.16/10.65.1.16:2181, initiating session
     [INFO ] 2018-04-26 13:09:16 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1299):Session establishment complete on server 10.65.1.16/10.65.1.16:2181, sessionid = 0x162b283c17b0032, negotiated timeout = 30000
     [INFO ] 2018-04-26 13:09:16 org.I0Itec.zkclient.ZkClient@(ZkClient.java:449):zookeeper state changed (SyncConnected)
     [WARN ] 2018-04-26 13:15:43 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server 10.65.1.16/10.65.1.16:2181, unexpected error, closing socket connection and attempting reconnect
     java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:15:43 org.I0Itec.zkclient.ZkClient@(ZkClient.java:449):zookeeper state changed (Disconnected)
     [INFO ] 2018-04-26 13:15:45 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [WARN ] 2018-04-26 13:15:45 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server null, unexpected error, closing socket connection and attempting reconnect
     java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:15:47 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [WARN ] 2018-04-26 13:15:47 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server null, unexpected error, closing socket connection and attempting reconnect
     java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:15:48 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [WARN ] 2018-04-26 13:15:48 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server null, unexpected error, closing socket connection and attempting reconnect
     java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:15:49 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [WARN ] 2018-04-26 13:15:49 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server null, unexpected error, closing socket connection and attempting reconnect
     java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:15:51 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [WARN ] 2018-04-26 13:15:51 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1162):Session 0x162b283c17b0032 for server null, unexpected error, closing socket connection and attempting reconnect
     java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[INFO ] 2018-04-26 13:15:53 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1032):Opening socket connection to server 10.65.1.16/10.65.1.16:2181. Will not attempt to authenticate using SASL (unknown error)
     [INFO ] 2018-04-26 13:15:53 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:876):Socket connection established to 10.65.1.16/10.65.1.16:2181, initiating session
     [INFO ] 2018-04-26 13:15:53 org.apache.zookeeper.ClientCnxn$SendThread@(ClientCnxn.java:1299):Session establishment complete on server 10.65.1.16/10.65.1.16:2181, sessionid = 0x162b283c17b0032, negotiated timeout = 30000
     [INFO ] 2018-04-26 13:15:53 org.I0Itec.zkclient.ZkClient@(ZkClient.java:449):zookeeper state changed (SyncConnected)
     